dace/frontend/python/parser.py

```diff
@@ -686,7 +686,10 @@ def _generate_pdp(self, args, kwargs, simplify=None) -> SDFG:
             self.global_vars[self.objname] = self.methodobj
 
         for k, v in argtypes.items():
-            if v.transient:  # Arguments to (nested) SDFGs cannot be transient
+            if isinstance(v, data.View):  # Arguments to (nested) SDFG cannot be Views
```

Comment:
Yes, because there is no code that goes over all the nodes of the connector-view in the nested SDFG to add 'views' connectors. There are a few issues here. First, such code would be slow. Second, I am not sure how we would define the "view" direction in such cases. The connector is a "View" of the parent SDFG data (not of some nested SDFG transient), which is anyway handled in code generation by representing the nested SDFG as a function call. Are there any actual cases where we want nested SDFG connectors to be Views? It looks to me that this is/was a (so far) benign side-effect rather than something intended.